src/components/ImageUploadForm.vue

```diff
@@ -57,7 +57,7 @@
       ></v-text-field>
 
       <h3 class="mt-5 mb-2 primary--text">我要通報...</h3>
-      <v-select :items="reportTypeItems" v-model="formState.reportType" solo outlined placeholder="未選擇" />
+      <v-select :items="reportTypeItems" v-model="formState.type" solo outlined placeholder="未選擇" />
```

Comment:
This fixes the None type of a factory object.

Reply:
Oh I suppose it's just a property name. And 'reportType' is more specific than 'type', I think.
So it breaks code somewhere because the name is different?
I can't test this out because my local backend server can't start currently for some weird issue.
Anyway, I think keeping property name to 'type' for now is fine 😄

Reply:
Right, the name matters.

I am very new to Vue. I *guess* here explains:
- What is `formState` object? It is returned by `createFactoryFormState` See https://github.com/tai271828/disfactory-frontend/blob/main-tbbca/src/components/CreateFactorySteps.vue#L362 and https://github.com/tai271828/disfactory-frontend/blob/main-tbbca/src/components/CreateFactorySteps.vue#L180
- What is `createFactoryFormState`? https://github.com/tai271828/disfactory-frontend/blob/main-tbbca/src/components/CreateFactorySteps.vue#L261 then the name `type` is definitive along with `createFactoryFormState`.